### PR TITLE
Added the feature to auto comment on a new issue creation 

### LINKED ISSUE #305: Feature for auto commenting on a new issue creation
I would like an automated feature that posts a predefined comment on an issue as soon as it is created. This feature aims to address the problem of delayed and inconsistent communication following the creation of issues. By automatically commenting on issues as soon as they are created, it ensures that contributors receive immediate feedback and acknowledgment for their efforts. This fosters a positive and supportive environment for collaboration, encouraging continued participation and engagement from contributors.

Could you please assign me this issue under GSSOC'24.

### PR DESCRIPTION
**Pull request by suhanipaliwal**
Fixes #305

### CODE REVIEW

**Comment by abhay-raj19** [comment]
Can you put up the video recording of the local working.

**Comment by abhay-raj19** [comment]
@suhanipaliwal ????

**Comment by abhay-raj19** [comment]
No changes up till now @suhanipaliwal , are you working on it or not ?

**Comment by suhanipaliwal** [comment]
Please merge this pull request and add labels as per gssoc guidelines.

File: .github/workflows/auto-comment-on-issue.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            -H "Accept: application/vnd.github.v3+json" \
+            https://api.github.com/repos/${{ github.repository }}/issues/${{ github.event.issue.number }}/comments \
+            -d "$COMMENT")
+          cat response.json
+          if [ "$RESPONSE" -ne 201 ]; then
+            echo "Failed to add comment"
+            exit 1
+          fi
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (abhay-raj19) [inline .github/workflows/auto-comment-on-issue.yml:35]: There is a specific convention required like you can not explicitly define the secret via GitHub as the starting name of the token.

Comment (abhay-raj19) [reply .github/workflows/auto-comment-on-issue.yml:35]: Just make sure from next time onwards it , not happen you can start with any word just skip the GitHub.

**Comment by abhay-raj19** [comment]
LGTM @suhanipaliwal

**Pull request merged**